### PR TITLE
feat: Load complete event on login of user

### LINKED ISSUE #205: App crash if attendees are saved before complete event
As attendee rely on the ticket to be foreign key, Foreign Key Constraint fails when attendees are saved before tickets( included in complete event )

We have to design a synchronisation mechanism to ensure this doesn't happen

### PR DESCRIPTION
**Pull request by iamareebjamal**
Fixes #205 

- Load the complete event with the partial event
- Add corresponding test

### CODE REVIEW

**Comment by iamareebjamal** [comment]
@mejariamol @rohanagarwal94 @arp95 @Shailesh351 Please review

File: app/src/main/java/org/fossasia/openevent/app/data/EventRepository.java [hunk 1]
```diff
@@ -110,9 +110,21 @@ public EventRepository(IUtilModel utilModel, IDatabaseRepository databaseReposit
...
+                    databaseRepository.saveList(Event.class, events)
                     .subscribe())
-                .flatMapIterable(events -> events));
+                .flatMapIterable(events -> events))
+                .doOnEach(eventNotification -> {
+                    // Download all complete events in one go
+                    if (!eventNotification.isOnNext())
+                        return;
+                    Event event = eventNotification.getValue();
+                    getEvent(event.getId(), false)
```
Comment (mejariamol) [inline app/src/main/java/org/fossasia/openevent/app/data/EventRepository.java]: Please add check here if an event is incomplete.

Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/data/EventRepository.java]: getEvent always returns complete event. Checking completeness is redundant

Comment (mejariamol) [inline app/src/main/java/org/fossasia/openevent/app/data/EventRepository.java]: I was saying before calling `getEvent`. But I think its not required now. What do you say?

**Pull request merged**